images/img-builder.sh

```diff
@@ -32,7 +32,6 @@ mkdir -p root/grub2/x86_64-efi
 mkdir -p root/cOS
 cp recovery/usr/share/grub2/x86_64-efi/*.mod root/grub2/x86_64-efi
 cp recovery/etc/cos/grub.cfg root/grub2
-sed -i 's/${saved_entry}/recovery/g' root/grub2/grub.cfg
```

Comment:
umm, I missed this but doesnt this means that the raw image will try to boot to `cos` directly and fail? even if we run user data, we need to boot into a "working" system to deploy that data...

Reply:
ah I see, it will keep trying until it falls under cos recovery :+1: